### PR TITLE
Feature lwdev-4336 logging for fee

### PR DESCRIPTION
**Pull request by MonarchL**

### CODE REVIEW

File: src/Lykke.Job.TransactionHandler/Queues/TransferQueue.cs [hunk 1]
```diff
@@ -106,7 +110,9 @@ public void Stop()
 
         public async Task ProcessMessage(TransferQueueMessage queueMessage)
         {
-            var amount = queueMessage.Amount.ParseAnyDouble();
+            var logTask = _transferLogRepository.CreateAsync(queueMessage.Id, queueMessage.Date, queueMessage.FromClientId, queueMessage.ToClientid, queueMessage.AssetId, queueMessage.Amount, queueMessage.FeeSettings.ToJson(), queueMessage.FeeData.ToJson());
```
Comment (starkmsu) [inline src/Lykke.Job.TransactionHandler/Queues/TransferQueue.cs]: Possible NRE on Fee fields?

File: src/Lykke.Job.TransactionHandler/Queues/TransferQueue.cs [hunk 2]
```diff
@@ -106,7 +110,9 @@ public void Stop()
...
+
+            var amount = queueMessage.Amount.ParseAnyDouble() - queueMessage.FeeData.Amount.ParseAnyDouble();
```
Comment (starkmsu) [inline src/Lykke.Job.TransactionHandler/Queues/TransferQueue.cs]: What happens if FeeData is null?

File: src/Lykke.Job.TransactionHandler/Queues/TransferQueue.cs (hunk 1 above)
Comment (MonarchL) [reply src/Lykke.Job.TransactionHandler/Queues/TransferQueue.cs]: fixed

File: src/Lykke.Job.TransactionHandler/Queues/TransferQueue.cs (hunk 2 above)
Comment (MonarchL) [reply src/Lykke.Job.TransactionHandler/Queues/TransferQueue.cs]: fixed

**Pull request merged**